addons/website_sale/models/product_template.py

```diff
@@ -74,7 +74,10 @@ def _set_base_unit_id(self):
     @api.depends('price', 'list_price', 'base_unit_count')
     def _compute_base_unit_price(self):
         for template in self:
-            template.base_unit_price = template.base_unit_count and (template.price or template.list_price) / template.base_unit_count
+            if hasattr(template.id, 'ref') and not template.id.ref:
```

Comment:
I think `if template.id:` should be sufficient.

You can probably use a temporary var for more readability.
(e.g. `template_price = (template.price or template.list_price) if template.id else template.list_price`)